src/handlers/security-headers-handler.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  securityHeadersHandler: (req, res, next) => {
+    res.set({
+      "Strict-Transport-Security": "max-age=31536000",
+      "Content-Security-Policy": "default-src 'self';",
+      "X-Frame-Options": "DENY",
+      "X-XSS-Protection": "0",
```

Comment:
I think this should be 1 to enable it?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection

Reply:
You're right
> The X XSS Protection header is designed to detect and assist in  protecting against cross-site scripting (“XSS”) attacks. However,  most modern browsers ignore this header now as having it on  can introduce additional security issues. The header should  therefore be set to off.